apps/preview/nuxt/pages/showcases/Combobox/ComboboxBasic.vue

```diff
@@ -0,0 +1,263 @@
+<template>
+  <div ref="referenceRef" class="relative">
+    <label :class="['font-medium typography-label-sm', { 'text-disabled-900': isDisabled }]" :for="id"> Country </label>
+    <SfInput
+      :id="id"
+      ref="inputRef"
+      v-model="inputModel"
+      role="combobox"
+      aria-label="Choose country"
+      placeholder="Choose country"
+      :aria-controls="listId"
+      aria-autocomplete="both"
+      :aria-disabled="isDisabled"
+      :aria-expanded="isOpen"
+      :aria-activedescendant="currentFocus?.id"
+      :invalid="isValid === false && !isOpen"
+      :disabled="isDisabled"
+      :class="[
+        'cursor-pointer',
+        {
+          '!text-disabled-500': isDisabled,
+        },
+      ]"
+      :wrapper-class="{
+        '!bg-disabled-100 !ring-disabled-300 !hover:ring-disabled-300 !ring-1': isDisabled,
+      }"
+      @blur="handleBlur"
+      @focus="isValid = undefined"
+      @click="toggle()"
+      @keydown="handleInputKeyDown"
+      ><template #suffix>
+        <SfIconExpandMore
+          :class="[
+            'ml-auto text-neutral-500 transition-transform ease-in-out duration-300',
+            {
+              'rotate-180': isOpen,
+              '!text-disabled-500 cursor-not-allowed': isDisabled,
+            },
+          ]"
+          @click="!isDisabled && toggle()"
+        />
+      </template>
+    </SfInput>
+    <div v-if="isOpen" ref="floatingRef" :style="style" class="left-0 right-0">
+      <div
+        v-if="isLoadingSnippets"
+        class="flex items-center justify-center w-full h-20 py-2 bg-white border border-solid rounded-md border-neutral-100 drop-shadow-md"
+      >
+        <SfLoaderCircular />
+      </div>
```

Comment:
Loading indication was removed from the scope of this task and the designs

Reply:
Thx, I removed it.